qemu/tests/timedrift_with_stop.py

```diff
@@ -53,7 +53,10 @@ def run(test, params, env):
     stop_iterations = int(params.get("stop_iterations", 1))
     stop_time = int(params.get("stop_time", 60))
     stop_with_signal = params.get("stop_with_signal") == "yes"
+    ntp_stop_cmd = params["ntp_stop_cmd"]
 
+    logging.info("Stop ntp service in guest")
+    session.cmd(ntp_stop_cmd)
```

Comment:
it had better add a check point for this command ?

Reply:
Test result

Host_RHEL.m8.u5.product_rhel.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.timedrift.shared_ntp_date.with_stop.clock_host: PASS
Host_RHEL.m8.u5.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.5.0.ppc64le.io-github-autotest-qemu.timedrift.shared_ntp_date.with_stop.clock_vm: PASS